src/nci_adapter.c

```diff
@@ -397,6 +521,46 @@ nci_adapter_create_known_tag(
     return NULL;
 }
 
+static
+NfcTag*
+nci_adapter_create_other_tag(
+    NciAdapter* self,
+    const NciIntfActivationNtf* ntf)
+{
+    const NciModeParam* mp = ntf->mode_param;
+    NfcParamPoll poll;
+
+    /* Figure out what kind of target we are dealing with */
+    switch (ntf->mode) {
+    case NCI_MODE_PASSIVE_POLL_A:
+        if (mp) {     /* Caller didn't check it for NULL */
```

Comment:
The common practice in this code is to add `/* Caller checked it for NULL */` kind of comment where a pointer **is not** checked for NULL. Checking for NULL is considered a normal practice not requiring any comments.

Reply:
Thanks, fixed